vm/sdk/src/wallet.rs

```diff
@@ -11,6 +11,17 @@ pub struct SpendArguments {
   pub amount: u64,
 }
 
+#[derive(Clone, Debug, Encode, Decode, PartialEq, Eq)]
+pub struct VerifyArguments {
+  pub raw: Vec<u8>,
+  pub sig: [u8; 64],
+}
+
+pub fn encode_verify_inner(raw: Vec<u8>, sig: [u8; 64]) -> Vec<u8> {
+  let args = VerifyArguments { raw, sig };
+  args.encode()
+}
```

Comment:
Two comments. 

From the perspective of SCALE encoding, the `VerifyArguments` structure is equivalent to a tuple `(Vec<u8>, [u8; 64])` so a custom type to be used only in this one method isn't needed. This is equivalent: 
```suggestion
pub fn encode_verify_inner(raw: Vec<u8>, sig: [u8; 64]) -> Vec<u8> {
  (raw, sig).encode()
}
```

Another thing is that it is weird to have a public `*_inner()` function.